### ISSUE TITLE
New eyes

### ISSUE THREAD

**Issue by android272** [teeworlds/teeworlds#1391]
I know that the new skin system is already complete or at least very close to being complete but I wanted to give my input on the new eyes.  It is an excellent system if everyone uses the stranded eyes but not everyone does.  I have recently created my own skin of a character from my comic book I wrote in middle and high school.  This character just happens to look like Tees even though I did not know about TeeWorlds before hand.  Well my character does not have the standard black eyes like most tees do but are a bit more anime style. 

Here are what my characters eyes look like.
![g6364](https://cloud.githubusercontent.com/assets/5912527/9570891/21edb75e-4f63-11e5-81db-245a31267680.png)

Here is what I propose the new eyes look like. 

![g6275](https://cloud.githubusercontent.com/assets/5912527/9570892/234502e2-4f63-11e5-8f6c-6d5f333d1b10.png)

So just like the body has the bold outline, body, shadow and thin outline these eyes would have the bold outline, the eye color(what would be changed if the player wants to change the color of their characters eyes, and decorations on top of the eye.  for my eyes I have little bubbles as if light was shining in the tees eyes like in an anime.

oh and the sixth set of eyes are for when Tees blink. I think they should blink ever few seconds regardless of whether they are getting shot, or someone gets the katana.

**Comment by ObaniGemini** [comment]
I prefer the classical and simple toon-look personally :)

**Comment by TheRealOrangus** [comment]
> I prefer the classical and simple toon-look personally :)

You're not alone.

**Comment by ObaniGemini** [comment]
Maybe could you make your work and propose it in a pull request for we can use it as an optionnal feature ?(in parameters)

**Comment by android272** [comment]
With the new skin system this would just be like changing your body or your "tattoos".  If you want the classical black eyes you can use them and they would still be the default.  But if people want to change the color of their tees eyes then they can change the eyes.

**Comment by android272** [comment]
Just like the body only the middle section of these eyes would be affected by color.  So the eyebrows/light highlights to the left and the eye outline to the right would remain black while only the eye color in the middle would be affected by color.  This is the same as the body. Only the body color changes color while the outline and the over line remain black(or what ever color they are in the skin).

**Comment by android272** [comment]
Here are a few variations on the eyes.

![eyes](https://cloud.githubusercontent.com/assets/5912527/11314089/336ab9b8-8fb0-11e5-9db3-6a77531eba83.png)

**Comment by ObaniGemini** [comment]
Did you seriously tried it in-game ?

**Comment by android272** [comment]
Yes the skin that I use have the blue eyes and it looks good.  It is a bit weird as I am playing on 0.6 and the way that it does the eyes it copies and flips the eye.  so the right eyes light bobbles were off.  I did not try it on 0.7 which has its own sprite for the eyes and show both eyes.  It would look more like the images above in 0.7.

**Comment by android272** [comment]
Here are two 0.6 skins for people to test with and some eyes if you are playing 0.7.  I have not tested the 0.7 eyes as I don't know how to build it on linux. 


![black_eyes](https://cloud.githubusercontent.com/assets/5912527/11319503/524dfb8c-9047-11e5-9e25-493c644ce413.png)
![blue_eyes](https://cloud.githubusercontent.com/assets/5912527/11319504/524fb256-9047-11e5-826a-c89690cbc7dd.png)

![black_eyes](https://cloud.githubusercontent.com/assets/5912527/11319515/eafe1254-9047-11e5-8a3b-dcb06c59d2ca.png)
![blue_eyes](https://cloud.githubusercontent.com/assets/5912527/11319518/eb00d43a-9047-11e5-9ef2-76953e3897a9.png)
![brown_eyes](https://cloud.githubusercontent.com/assets/5912527/11319517/eb00c6ca-9047-11e5-86e1-c20469c6ead4.png)
![green_eyes](https://cloud.githubusercontent.com/assets/5912527/11319519/eb01b6c0-9047-11e5-9e96-9ad658940869.png)
![red_eyes](https://cloud.githubusercontent.com/assets/5912527/11319516/eaff261c-9047-11e5-9450-c3a480775150.png)
